### PR TITLE
Go mod ify

### PR DESCRIPTION
**Pull request by dizveloper**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module github.com/sygaldry/sygaldry
+
+go 1.13
+
+require (
```
Comment (bricerising) [inline go.mod:5]: Is this an option for documenting dependencies that's different from the deps project?

Comment (dizveloper) [reply go.mod:5]: This makes deps unnecessary. It's autogenerated when you "go mod init" and then anything you "go get" gets added

Comment (theinfamousrj) [reply go.mod:5]: Does it add to a doc somewhere? I can add link to that in our docs! 😃

Comment (dizveloper) [reply go.mod:5]: I don't know about a doc, but it adds everything to the go.mod file

**Pull request merged**